### PR TITLE
Several fixes.

### LINKED ISSUE #498: Bug in rule “headers.dl”: Bad message for This Version syntax
It says "Wrong syntax for This Version link. Use a 'https' link.", but https://api.csswg.org/bikeshed/?url=https://raw.githubusercontent.com/w3c/permissions/01df4eaa22343b50c9ee8a38653d17f982cea659/index.bs has a "This version" link of https://w3c.github.io/permissions/, which *does* start with 'https'.

Found [while checking `https://api.csswg.org/bikeshed/?url=https://raw.githubusercontent.com/w3c/permissions/01df4eaa22343b50c9ee8a38653d17f982cea659/index.bs`](https://www.w3.org/pubrules/?url=https%3A%2F%2Fapi.csswg.org%2Fbikeshed%2F%3Furl%3Dhttps%3A%2F%2Fraw.githubusercontent.com%2Fw3c%2Fpermissions%2F01df4eaa22343b50c9ee8a38653d17f982cea659%2Findex.bs&profile=WD-Echidna&validation=simple-validation&noRecTrack=false&informativeOnly=false&echidnaReady=true&patentPolicy=pp2004).

### LINKED ISSUE #505: Uncaught TypeError in rule “structure/name”
```
lib/rules/structure/name.js:37
                    if (!result1.ok || !result2.ok || result1.text !== result2.text) {
                                               ^

TypeError: Cannot read property 'ok' of undefined
    at lib/rules/structure/name.js:37:48
    at Request.callback (node_modules/superagent/lib/node/index.js:587:12)
    at ClientRequest.<anonymous> (node_modules/superagent/lib/node/index.js:539:10)
    at emitOne (events.js:96:13)
    at ClientRequest.emit (events.js:188:7)
    at TLSSocket.socketErrorListener (_http_client.js:306:9)
    at emitOne (events.js:96:13)
    at TLSSocket.emit (events.js:188:7)
    at connectErrorNT (net.js:1015:8)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
```

Will happen, for example, if there's a network hiccup, or if `<URL>/Overview.html` fails or can't be resolved for any reason. The app crashes.

### PR DESCRIPTION
**Pull request by tripu**
* Fixes #498.
* Fixes #505.
* Another small fix.

--

Should this be 3 PRs instead? Sorry, I realised too late&hellip; (Although grouping them saves overhead!)

### CODE REVIEW

File: lib/l10n-en_GB.js [hunk 1]
```diff
@@ -40,13 +40,13 @@ exports.messages = {
 ,   "headers.dl.this-link":                 "Link href and text differ for This Version."
 ,   "headers.dl.this-date":                 "Mismatch between document date and This Version link."
 ,   "headers.dl.no-date":                   "Cannot find document date."
-,   "headers.dl.this-syntax":               "Wrong syntax for This Version link. Use a 'https' link."
+,   "headers.dl.this-syntax":               "Wrong syntax for This Version link."
```
Comment (jyasskin) [inline lib/l10n-en_GB.js]: Having a URL that points outside the https://www.w3.org/TR/ space isn't the "wrong syntax", it's a disallowed URL.

File: public/js/specberus.js [hunk 2]
```diff
@@ -241,8 +241,8 @@ jQuery.extend({
             ,   patentPolicy = $patentPolicy.find('label.active').attr('id')
             ;
             profile = $profile.val();
-            if (!url) addMessage(MSG_ERROR, 'Missing “URL” parameer');
-            if (!profile) addMessage(MSG_ERROR, 'Missing “profile” parameter');
+            if (!url) addMessage(MSG_ERROR, {message: 'Missing “URL” parameer'});
```
Comment (jyasskin) [inline public/js/specberus.js]: FWIW, "parameer" is spelled wrong.

File: lib/l10n-en_GB.js (hunk 1 above)
Comment (tripu) [inline lib/l10n-en_GB.js]: I'm using the wording of [pubrules](https://www.w3.org/pubrules/doc/rules/?profile=WD#docIDThisVersion).

File: public/js/specberus.js (hunk 2 above)
Comment (tripu) [inline public/js/specberus.js]: [Fixed](https://github.com/w3c/specberus/pull/506/commits/9ad0d34c7dfd2e8aa99c39cea7866760d0430d33), thank you!

**Pull request merged**